psignifit/psigniplot.py

```diff
@@ -49,8 +48,8 @@ def plot_psychometric_function(result: Result,  # noqa: C901, this function is t
     if plot_data:
         y_data = data[:, 1] / data[:, 2]
         # the size is proportional to the sqrt of the data size, as in the MATLAB version.
-        # We added a factor of 1000 to make visually similar to the MATLAB version
-        size = np.sqrt(_data_size / data[:, 2])*(data_size*1000)
+        # We added a factor of 100 to make visually similar to the MATLAB version
+        size = np.sqrt(data[:, 2])*(data_size*100)
```

Comment:
As far as I know, the factor of 100-1000 was always there, and recently @guillermoaguilar added a `data_size` option in case this default is not good enough. Maybe we should move the factor directly in the keyword argument? I.e. make `data_size = 100`  by default, and then the user can adjust it if necessary

Reply:
oh but I see now the "reversing" of the logic... it's true, it seems the wrong way around

Reply:
`data_size` and `_data_size` were two different values though. That was causing the problem.

Reply:
true, sorry I missed that

Reply:
its true that we could abandon the 100 scaling factor... on the other hand its also intuitive to have the input value 1 be a reasonable size already. I dont mind either way.

Reply:
no need for changes, afaic